src/actions/socialAuth.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+import { SOCIAL_LOGIN_SUCCESS, SOCIAL_LOGIN_FAILURE } from './type';
+
+export const loadUser = (token, username) => async (dispatch) => {
+  if (token) {
+    try {
+      localStorage.setItem('token', token);
+      localStorage.setItem('username', username);
```

Comment:
why are you setting a username to local storage yet you are also dispatching it to the store?